validator/client/service_test.go

```diff
@@ -33,22 +33,6 @@ func TestStop_CancelsContext(t *testing.T) {
 	}
 }
 
-func TestLifecycle(t *testing.T) {
```

Comment:
This test was basically useless. Testing `Start()` stopped at https://github.com/prysmaticlabs/prysm/blob/3b69f7a196755a70bf59a9462b79c6657781615f/validator/client/service.go#L140